src/components/WorkspaceMosaic.js

```diff
@@ -69,10 +69,24 @@ export class WorkspaceMosaic extends React.Component {
     const window = windows[id];
     if (!window) return null;
     return (
-      <Window
-        key={window.id}
-        window={window}
-      />
+      <MosaicWindow
+        toolbarControls={[]}
+        additionalControls={[]}
+        path={path}
+        windowId={window.id}
+        renderPreview={() => (
+          <div className="mosaic-preview">
+            <div className="mosaic-window-body">
+              <h4>Mirador</h4>
```

Comment:
Can we i18n this?

Reply:
Good call. Just pushed a commit to do this.